src/python/nimbusml/pipeline.py

```diff
@@ -1816,22 +1819,43 @@ def _predict(self, X, y=None,
                 isinstance(X, DataFrame) and isinstance(y, (str, tuple))):
             y = y_temp
 
+        model_zip = ZipFile(self.model)
```

Comment:
Should this be a `with` statement so that `model_zip` is closed after the next line executes? Or, should `model_zip` be closed after it is used?

Reply:
good point, made a change

---
In reply to: [316409298](https://github.com/microsoft/NimbusML/pull/230#discussion_r316409298) [](ancestors = 316409298)